<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialServer</class>
 <widget class="QDialog" name="SerialServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="PORT_NUM_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>32</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QComboBox" name="LST_PORT_TYPE_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="LST_PORT_TYPE_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="PORT_NUM_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>62</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="LST_PORT_TYPE_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="PORT_NUM_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>92</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QComboBox" name="LST_PORT_TYPE_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="PORT_NUM_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>122</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="IP">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>162</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EDT_IP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
